### ISSUE TITLE
opm index add fails on s390x as binary was compiled with 'CGO_ENABLED=0'

### ISSUE THREAD

**Issue by pierswilliams** [operator-framework/operator-registry#510]
When running `opm index add` on s390x, using `registry.redhat.io/openshift4/ose-operator-registry:v4.6`as the value for the `--binary-image`parameter, builds are failing with the following:

```
Error: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work.
```

This command worked as expected using `v4.5` of OSE Operator Registry. **Also an issue on current upstream OPM version 1.15.1**

**Comment by katheris** [comment]
I also see this error when running using the darwin release. It is preventing us from using opm.

**Comment by tegioz** [comment]
If you have Go installed, you can build it yourself as a temporary solution @katheris.

```
GO111MODULE=on go get github.com/operator-framework/operator-registry/cmd/opm@v1.15.1
```

This will build and install `opm` in $GOPATH/bin. It works fine for me in macOS.

**Comment by katheris** [comment]
@tegioz I had a go at running with the command you suggested but then got this error:
```$ GO111MODULE=on go get github.com/operator-framework/operator-registry/cmd/opm@v1.15.1
go: found github.com/operator-framework/operator-registry/cmd/opm in github.com/operator-framework/operator-registry v1.15.1
# github.com/operator-framework/operator-registry/cmd/opm
/usr/local/Cellar/go/1.15.2/libexec/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
ld: sectionForAddress(0x13208BE) address not in any section file '/var/folders/ft/8fvxbf391j3_gbwkcsm2mj5w0000gn/T/go-link-059007946/go.o' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

A quick search found this issue: https://github.com/golang/go/issues/28704 which suggested the fix is to set `CGO_ENABLED=0` which is the original issue shown.

I wonder if I am running with the wrong version of go? What version do you have?

**Comment by tegioz** [comment]
You may be hitting [this issue](https://github.com/golang/go/issues/40974) @katheris, which I think was addressed in Go 1.15.4.

I'm using 1.15.5 right now, but I'm it's possible that I had 1.15.4 the day I installed it.

**Comment by saada** [comment]
Only thing that fixed it for me was upgrading opm to 1.15.2. Thanks!

**Comment by katheris** [comment]
Useful to see that this is hopefully fixed in 1.15.2! We ended up building it ourselves, we had to build separate versions on each of the operating systems we wanted to run it on.

**Comment by ecordell** [comment]
This appears fixed in 1.15.2. Please re-open if I have misunderstood. 

It's worth mentioning that we can no longer support upstream releases of s390x and z. The makefiles should work on those systems if they need to be built there, or the downstream releases from Red Hat will work.
